services/controllerhost/replicaStatsMRUCache.go

```diff
@@ -77,47 +78,51 @@ func (c *replicaStatsMRUCache) get(extent extentID, store storeID) (val *shared.
 	}
 
 	// Check for cache hit
-	if psmap, ok = c.cache[extent]; ok {
-		if val, ok = psmap.m[store]; ok { // store == `` will fall through here
-			if val != nil {
-				return
-			}
-		}
-
-		// Get any one store's replica stats; see if it is sealed
-	oneStore:
-		for foundStore, val = range psmap.m {
-			if val != nil {
-				break oneStore
-			}
-		}
+	intrfc = c.cache.Get(string(extent))
```

Comment:
This isn't a problem. It is true that we have merged extents that exist in two destinations, but they are deleted in one of them. putPlaceholders ensures that each destination gets the proper view of the extent (i.e. different singleCGVisibility), even though the underlying replica stats doesn't change. 

So merge doesn't cause a problem here. We don't calculate queue depth for consumer groups of the DLQ, either. Nor do we calculate queue depth for the DLQ destination itself.